<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logoff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="fundo">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-5</y>
      <width>410</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(192, 255, 199, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="tela_inicial">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>-5</y>
       <width>410</width>
       <height>510</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(43, 43, 43);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="inicial_titulo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>400</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(31, 106, 165);
background-color: rgba(43, 43, 43, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;SAIR DO WINDOWS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QFrame" name="inicio_quadro">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>150</y>
        <width>351</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 33, 33);
border-radius:20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="quadro_pri">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>131</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:120px;
font-family: arial</string>
       </property>
       <property name="text">
        <string>01</string>
       </property>
      </widget>
      <widget class="QLabel" name="quadro_sec">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>131</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:120px;
font-family: arial</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
      <widget class="QLabel" name="extras_7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>21</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:120px;
font-family: arial</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLabel" name="extras_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>67</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:25px;
font-family: arial</string>
       </property>
       <property name="text">
        <string>Horas</string>
       </property>
      </widget>
      <widget class="QLabel" name="extras_9">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>200</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:25px;
font-family: arial</string>
       </property>
       <property name="text">
        <string>Minutos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="up_pri">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 35px</string>
       </property>
       <property name="text">
        <string>⮝</string>
       </property>
      </widget>
      <widget class="QPushButton" name="up_sec">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:35px</string>
       </property>
       <property name="text">
        <string>⮝</string>
       </property>
      </widget>
      <widget class="QPushButton" name="down_pri">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:35px</string>
       </property>
       <property name="text">
        <string>⮟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="down_sec">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:35px</string>
       </property>
       <property name="text">
        <string>⮟</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="modo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 106, 165);
color: white; 
border-radius: 7px;
font-size: 15px;</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="maxVisibleItems">
       <number>2</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Desligar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reiniciar</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_iniciar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>450</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 106, 165);
color: rgb(255, 255, 255); 
border-radius: 10px;
font-weight: bold;
font-size:15px;</string>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Em quanto tempo deseja&lt;br/&gt;Sair do windows?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../AppData/Local/Programs/Python/Python311/Lib/site-packages/qt6_applications/Qt/bin/icons.qrc"/>
 </resources>
 <connections/>
</ui>
